### ISSUE TITLE
[Bug] /tell-wr crashes game

### ISSUE THREAD

**Issue by DeafChips** [codecat/tm-better-chat#42]
During today's TOTD (March 31st, 2022), using the `/tell-wr` command crashes the game. No logs are printed before game crash. Not sure if it's an API update or possibly related to NadeoServices dependency.

**Comment by codecat** [comment]
I can't reproduce this:

![image](https://user-images.githubusercontent.com/136534/161245476-d024d95a-c8e5-4543-a12b-fc8cae49c111.png)

**Comment by DeafChips** [comment]
You're right, it seems to be working now (not able to reproduce at this time). I raised the issue for visibility since it did occur for me and at least one other person yesterday. I will close and can re-open if the issue crops up again.

**Comment by codecat** [comment]
Please do! I'll keep trying as well in case I can reproduce it randomly.

**Comment by nbeerten** [comment]
I got the same bug today after using the `/wr` command, specifically in the knockouts of today's COTD. I have succesfully reproduced it by creating a new room with the "knockouts" mode, entering it and using the `/wr` command in that room. After pressing the `enter` key (sending the command) the whole game instantly freezes and closes. This seems to be consistent. Nothing to see in the logs either, as @chipsTM mentioned before.
